tests/unit_tests/chains/test_bash.py

```diff
@@ -0,0 +1,94 @@
+import subprocess
```

Comment:
for all these tests, its probably safer to use a temporary directory to avoid any unexpected side effects

https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

Reply:
Thank you!